docs/source/testing-and-issue-tracking/Kokkos-Project-Planning.md

```diff
@@ -0,0 +1,210 @@
+
+
+## Issue Prioritization
+
+Issue prioritization is performed via two avenues:
+- Kokkos Leadership meeting
+- General Kokkos developer meeting.
+
+The Leadership meeting happens every week on Mondays.
+It serves multiple purposes:
+- determine urgent action items for the week
+- go through new issue list, and triage criticality
+- work through Kokkos planning items
+- perform preliminary team assignments for new action items
+- generate a draft for the developer meeting agenda
+
+Prioritization of items is recorded in the [Kokkos project plan](https://github.com/orgs/kokkos/projects/1)
+
+Meeting notes are kept in a private repository: [internal repository](https://github.com/kokkos/internal-documents)
+
+Further issue prioritization happens at the developer meeting discussed below.
+
+## Developer Coordination
+
+The team primarily use the #nucleus channel on Slack to communicate.
+Members are added by Christian or Damien once they have joined [Slack](https://kokkosteam.slack.com).
+Developers can have both public and private conversations with each other.
+They can ask questions about parts of the code they are less familiar with or
+ask for feedback on any ongoing issue.
+Conversations on Slack are to be considered as ephemeral.  Messages older than 90 days are deleted (unpaid plan).
+If something needs to be referenceable longer term, then it needs to be discussed on GitHub wherever appropriate.
+Private information may be hosted on the [internal repository](https://github.com/kokkos/internal-documents) but do not post NDA data on there.
+
+Kokkos developer meeting held once a week on Wednesdays 2pm ET / 12 pm MT / 18:00 UTC on Zoom.
+The agenda is posted on the internal repository ahead of time (it can be found under the [`meeting-notes/`](https://github.com/kokkos/internal-documents/tree/master/meeting-notes/2023) directory).
+Developers are allowed to edit the agenda and add topics or issues that they would like to be discussed at the meeting.
+
+## Release Process
+
+The release process has six steps:
+
+- create release candidate branch
+- perform integration tests with release candidate
+- resolve issues and cherry-pick fixes to release candidate
+- check Changelog
+- tag a release
+- conduct release briefing for user community
+
+When nearing a desired release date, the release candidate branch will created from the Kokkos develop branch.
+Before creating the release candidate, possible delay reasons will be discussed at the developer meeting.
+This could include important bug fixes, or an important feature being in the last phase of code review,
+but is generally done under exceptional circumstances.
+Furthermore, major new features may be delayed for merging into the development branch, until after the creation
+of the release candidate.
+This ensures that major new features have a period of testing in the develop branch, before they are shipped.
```

Comment:
```suggestion
When nearing a desired release date, the release candidate branch will be created from the Kokkos develop branch.
Before creating the release candidate, possible delay reasons will be discussed at the developer meeting.
This could include important bug fixes, or an important feature being in the last phase of code review,
but is generally done under exceptional circumstances.
Furthermore, merging major new features into the development branch may be delayed until after the creation
of the release candidate.
This ensures that major new features have a period of testing in the develop branch before they are shipped.
```